backend/src/main/java/com/cs471/studentLoanSystem/sql/descriptions/Loan.java

```diff
@@ -0,0 +1,83 @@
+package com.cs471.studentLoanSystem.sql.descriptions;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+enum LoanStatus {
+    APPLIED,
+    APPROVED,
+    PAID
+}
```

Comment:
This should be moved into the Loan class so it becomes a class enum. It doesn't make sense that LoanStatus is outside

Reply:
Fixed, good catch